JustReleaseNotes/command_line.py

```diff
@@ -47,7 +47,13 @@ def generate_release_notes(configFile):
             directory = os.path.join(releaseNotesConfig["pathToSave"],packageName)
             conf["Source"]["Directory"] = directory
             repo = JustReleaseNotes.sourcers.factory.create(conf["Source"])
-            writer = JustReleaseNotes.writers.factory.create(conf["ReleaseNotesWriter"], ticketProvider)
+
+            writerTemplate = "${content}"
```

Comment:
What is the syntax of the template file ? Is it some well known templating language ?